druid-shell/examples/perftest.rs

```diff
@@ -14,7 +14,7 @@
 
 use std::any::Any;
 
-use time::get_time;
+use time::Time;
```

Comment:
So this is certainly a valid crate update, but I now think this is the wrong time type. It should probably be std::time::Instant, followed by `elapsed` to get the duration, then formatted through `Duration::as_secs_f64` (new as of Rust 1.38). The time crate has these same types and methods, but I don't think they add a lot of value over std.

Reworking this is probably out of scope for this PR, could be considered part of #476.

Reply:
sounds good.